substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -1946,6 +1949,9 @@ pub mod pallet {
 		NotMigrated,
 		/// This call is not allowed in the current state of the pallet.
 		NotSupported,
+		/// Account is blacklisted from participation in pools. This may happen if the account is
+		/// staking in another way already.
+		Blacklisted,
```

Comment:
As mentioned above, I think "BlackListed" may be too overwhelming for a end-user as it seems it's excluding the user from bonding/staking for malicious reasons. Maybe we could rename it to `UnableToStake` or something similar?

OTOH naming the config types as `Blacklisted` seems ok since that's internal to devs.